### PR TITLE
Helios.TextPad version 9.2.0

### LINKED ISSUE #101613: [Package Request]: TextPad
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.textpad.com/download
Publisher: Helios Software Solutions
Package Name: TextPad
Description: TextPad is a powerful, general-purpose editor for plain text files. Easy to use, with all the features a power user requires.
Package Version: 9.2.0
Installer URL: https://www.textpad.com/file?path=v9/setupv9.exe

### PR DESCRIPTION
**Pull request by afermar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
Resolves #101613 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101618)

### CODE REVIEW

File: manifests/h/Helios/TextPad/9.2.0/Helios.TextPad.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Helios.TextPad
+PackageVersion: 9.2.0
+Installers:
+- InstallerUrl: https://www.textpad.com/file?path=v9/setupv9.exe
+  Architecture: x64
+  InstallerType: exe
+  InstallerSha256: C43E7F36DB9C43F6856B7C651C2BC907899FA7962DA8AFEA6F8EA4149D2874B1
```
Comment (stephengillie) [inline manifests/h/Helios/TextPad/9.2.0/Helios.TextPad.installer.yaml:10]:
```suggestion
  InstallerSha256: 862441694DFC3DEE327103CF56FC3079924B0700750070E8EEB2DFBECA440C25
```

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 2:03:54 PM: 2023-04-06 14:03:02.894 [CLI ] Package hash verification failed. SHA256 in manifest [c43e7f36db9c43f6856b7c651c2bc907899fa7962da8afea6f8ea4149d2874b1] does not match download [862441694dfc3dee327103cf56fc3079924b0700750070e8eeb2

**Comment by BrandonWanHuanSheng** [comment]
> Manual Validation failed with:
> 
> > 2:03:54 PM: 2023-04-06 14:03:02.894 [CLI ] Package hash verification failed. SHA256 in manifest [c43e7f36db9c43f6856b7c651c2bc907899fa7962da8afea6f8ea4149d2874b1] does not match download [862441694dfc3dee327103cf56fc3079924b0700750070e8eeb2

This is normally a bots behind our way.

File: manifests/h/Helios/TextPad/9.2.0/Helios.TextPad.installer.yaml (hunk 1 above)
Comment (BrandonWanHuanSheng) [reply manifests/h/Helios/TextPad/9.2.0/Helios.TextPad.installer.yaml:10]: This sha256 above mismatch with my computer.  .2420BD2DFE2751E4F8DFFE6875D6026B62F6A90CEB142803A98323C2A6770631

**Comment by BrandonWanHuanSheng** [comment]
Searching for their location.

**Comment by BrandonWanHuanSheng** [comment]
@afermar Can we convert to draft because that URL is not support by winget? 
Screenshot (150)

**Comment by afermar** [comment]
Sure

**Comment by ItzLevvie** [comment]
[Policy] Area-External

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by Trenly** [comment]
Just testing something by commenting; Please ignore this

**Pull request closed without merging**